models/user.ts

```diff
@@ -0,0 +1,54 @@
+'use strict';
+import {
+  Model, UUIDV4
+} from 'sequelize'
+
+interface UserAttributes {
+  id: string
+  username: string
+  email: string
+  password: string
+}
+
+module.exports = (sequelize:any, DataTypes:any) => {
+  class User extends Model<UserAttributes> 
+    implements UserAttributes {
```

Comment:
Probably it's redundant to have "UserAttributes" in Model and again as implements.